### DISCUSSION TITLE
Image Field Is Not Working?

### DISCUSSION THREAD

**Discussion by Nick7961** [pods-framework/pods#6602] [Ideas]
### Description

Hi there...I'm working with the Pods & oxygen builder plugin to add custom fields in my Wordpress post type 
1 field is for banner
2 field is for link
3 field is for text

Everything is working properly with oxygen builder like links, and text but, somehow when I try to pull the banner image, it isn't displaying on the page...

And I try this with the ACF field it's working fine. Don't know what's the problem?

Please see the screenshot below... You can see that in HTML it should pass the URL in SRC but here is giving me SRC="array"...Please let me know how to fix this...

### Version

2.8.15

### Screenshots / Screencast

![Screenshot_21](https://user-images.githubusercontent.com/77339002/165947479-f2d6c474-4a09-4478-a0b2-19445adaafd1.png)
![Screenshot_20](https://user-images.githubusercontent.com/77339002/165947523-cdcde732-b1a4-4d4c-b325-18cbcb9067e3.png)

**Comment by JoryHogeveen** [comment]
It looks like the Oxygen builder isn't compatible with Pods.
@sc0ttkclark Maybe something for the page builder addon?

**Comment by sc0ttkclark** [comment]
How are you entering this field name? Try field_name.thumbnail

**Comment by Nick7961** [comment]
These are the field I've created...
![image](https://user-images.githubusercontent.com/77339002/166084565-65b4a9fa-9ce6-4f64-8d30-26901e3ddd49.png)

and as you've mentioned, "try field_name.thumbnail". I tried but still not working

**Comment by sc0ttkclark** [comment]
Sorry, I meant field_name._src.thumbnail

As per docs here: https://docs.pods.io/code/pods/field/field-notation-options/#File_Field_Image_URL_and_Sizing

**Comment by Nick7961** [comment]
I've tried all the methods from the documentation, and none of them seem to work... is it something with the dot (.) cuz I don't think I can add to the name field if I even try, it automatically adds underscore (_)

![image](https://user-images.githubusercontent.com/77339002/166087210-c729e549-3c3a-4361-b970-49058f0a7def.png)
![image](https://user-images.githubusercontent.com/77339002/166087216-c6bc593d-3f76-4544-a721-fe21983415f7.png)
![image](https://user-images.githubusercontent.com/77339002/166087238-8ad6d036-4042-42ab-82fc-9c2ea9fe42b3.png)
![image](https://user-images.githubusercontent.com/77339002/166087285-42a93876-43ed-45ca-8848-42f84b1b6c0d.png)
![image](https://user-images.githubusercontent.com/77339002/166087583-b9fb5529-a647-4775-9f4b-c3018de3d96f.png)
![image](https://user-images.githubusercontent.com/77339002/166087605-22db49d2-5d17-499c-b471-36cc6c8983ab.png)
